stdlib/public/SDK/Dispatch/Data.swift

```diff
@@ -114,10 +114,12 @@ public struct DispatchData : RandomAccessCollection, _ObjectiveCBridgeable {
 	///
 	/// - parameter buffer: The buffer of bytes to append. The size is calculated from `SourceType` and `buffer.count`.
 	public mutating func append<SourceType>(_ buffer : UnsafeBufferPointer<SourceType>) {
-		self.append(UnsafePointer(buffer.baseAddress!), count: buffer.count * sizeof(SourceType.self))
+    buffer.baseAddress!.withMemoryRebound(to: UInt8.self, capacity: buffer.count * sizeof(SourceType.self)) {
+		  self.append($0, count: buffer.count * sizeof(SourceType.self))
```

Comment:
Shouldn't this be `strideof`?  I see that the old code used `sizeof`, but it looks like a bug to me.